aws/resource_aws_autoscaling_group_test.go

```diff
@@ -1102,10 +1033,6 @@ func testAccCheckAWSAutoScalingGroupDestroy(s *terraform.State) error {
 
 func testAccCheckAWSAutoScalingGroupAttributes(group *autoscaling.Group, name string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
-		if *group.AvailabilityZones[0] != "us-west-2a" {
```

Comment:
This hardcoded AZ check is not needed because of the check that looks like this:

```go
tfawsresource.TestCheckTypeSetElemAttrPair("aws_autoscaling_group.bar", "availability_zones.*"...
```